### DISCUSSION TITLE
Pause the flow for manual input

### DISCUSSION THREAD

**Discussion by siarhei-zhmura** [postmanlabs/postman-flows#286] [General]
Hi all,

I wonder if it is achievable anyhow to pause/resume a flow and define a variable value for a particular step in the middle of the flow.
My case: I test e2e flow that uses web sms service for OTP auth. I'd like to get & use a real OTP sent to the mobile device during the test. But I'd like to save my time instead of setting a huge delay in advance.

Thanks

**Comment by superjose** [comment]
Thiiiiiiiiiiiiiiiiiiiiiiissss. It's my same problem!!! I'd love to have this!
